### PR TITLE
Teshari/Skrell food expansion

### PR DESCRIPTION
**Pull request by Greenjoe12345**
This PR adds a number of Sirisaian/Quer'balakian plants and foods, available to grow and cook, a number aimed towards teshari, and a number towards Skrell.
This will want a wiki update when it's in.
the recipes are as follows

DRINKS

Dyn tea = 1 dyn juice, 4 water = 5 dyn tea

Dyn iced tea = 1 dyn juice, 2 ice, 2 soda water = 5 dyn iced tea

FOOD

Dyn soup - MICROWAVE = 10 water, 2 dyn, 1 cabbage, 1 tomato

Zantiri - MICROWAVE = 10 water, 2 guami, 1 eki

Lortl - Oven = 1 q'lort, 1 salt = 3 lortl

Boiled fjylozyn - MICROWAVE = 1 fjylozyn, 5 water

Neaera stew - MICROWAVE = 10 dyn juice, 10 water, 2 guami, 2 eki, 1 Neaera meat

Candied Neaera eyes - OVEN = 5 cream, 1 dough, 1 neaera eyes = 6 candies

Neaera kebab - MICROWAVE = 1 Neaera meat, 1 Neaera kidney, 1 metal rod

Garani - FRYER = 2 boiled fjylozyn, 1 Neaera liver

Qa'zal bread - OVEN = 4 qa'zal dough

Grilled qa'zal flatbread - GRILL = 1 flat qa'zal dough

Baked ga'uli - OVEN = 1 ga'uli

Baked kirani - OVEN = 1 kirani

Stuffed ga'uli = OVEN = 1 ga'uli, 1 kirani, 1 Neaera meat

Kirani stew - MICROWAVE = 10 kirani jelly, 1 qa'zal, 1 Neaera meat

Qa'zal noodles - MICROWAVE = 1 kirani, 1 gu'ali, 3 Qa'zal dough strips

Kirani jellypuff - OVEN = 2 Qa'zal dough, 5 kirani jelly, 5 qa'zal flour

qa'zal dough = 10 qu'zal flour, 5 ga'uli juice

### CODE REVIEW

**Comment by Cerebulon** [comment]
Will also need review from skrellmaint who incidentally is also Ater so, handy!
Only notes from me:
"fjylozyn" does not really seem to match with skrell phonemes and could probably use a skrellier name to avoid confusion with sif style scandigarble
Not sure about All the exotic skrell seeds being available in seedvend? Up to ater.
You dont need to denote and date new additions to the food file. People have done it in the past but it seeeems to be a virgo thing or something and just needs cleaning up later? git blame does dating and attributing automatically and better anyway.
Also is this stuff From another server? Very mild concern about straight lore transplants. None of it seems conflicty its just a distinctiveness thing if the names/concepts are direct lifts.

**Comment by Greenjoe12345** [comment]
The stuff is from other servers, Yes, I did alter some stuff about them that wasn't fitting for here.
As for fjylozyn, I'll see about altering the name on that, maybe something like 'qa'lozyn'

File: code/modules/reagents/reagents/food_drinks.dm [hunk 1]
```diff
@@ -4464,3 +4464,59 @@
 	nutriment_factor = 2
 	glass_name = "kvass"
 	glass_desc = "A hearty glass of Slavic brew."
+
+//Querbalak stuff
+/datum/reagent/drink/juice/dyn
+	name = "Dyn juice"
+	id = "dynjuice"
+	description = "Juice from a Qerr'balakian herb. Good for you, but normally not consumed undiluted."
```
Comment (Atermonera) [inline code/modules/reagents/reagents/food_drinks.dm:4472]:
```suggestion
	description = "Juice from a Qerr'balakian herb. Good for you, but usually diluted for potability."

File: code/modules/reagents/reagents/food_drinks.dm [hunk 2]
```diff
@@ -4464,3 +4464,59 @@
...
+	taste_description = "astringent menthol"
+	color = "#00e0e0"
+	glass_name = "dyn juice"
+	glass_desc = "Juice from a Qerr'balakian herb. Good for you, but normally not consumed undiluted."
+	sugary = FALSE
+
+/datum/reagent/drink/tea/dyn
+	name = "Dyn tea"
+	id = "dynhot"
+	description = "An old-fashioned, but traditional skrellian drink with documented medicinal properties."
```
Comment (Atermonera) [inline code/modules/reagents/reagents/food_drinks.dm:4482]:
```suggestion
	description = "A traditional skrellian drink with documented medicinal properties."

File: code/modules/reagents/reagents/food_drinks.dm [hunk 3]
```diff
@@ -4464,3 +4464,59 @@
...
+	color = "#00e0e0"
+	taste_description = "fizzy mint tea"
+	allergen_type = null
+	glass_name = "dyn iced tea"
+	glass_desc = "A modern spin on an old formula. Good for you!"
+
+/datum/reagent/nutriment/qazal_flour
+	name = "Qa'zal flour"
+	id = "qazal_flour"
+	description = "Harvested from ground qa'zal, this is one of the main ingriedents in qa'zal bread."
```
Comment (Atermonera) [inline code/modules/reagents/reagents/food_drinks.dm:4504]:
```suggestion
	description = "Harvested from ground qa'zal, this is one of the main ingredients in qa'zal bread."

File: code/modules/reagents/reagents/food_drinks.dm [hunk 4]
```diff
@@ -4464,3 +4464,59 @@
...
+	name = "Kirani jelly"
+	id = "kirani_jelly"
+	description = "Sticky, sweet jelly from ground kiriani fruits."
+	taste_description = "ultra-sweet jelly"
+	color = "#993c5c"
+
+/datum/reagent/drink/gauli_juice
+	name = "Ga'uli juice"
+	id = "gauli_juice"
+	description = "Juice from a ga'uli pod, used within skrell and teshari cooking."
```
Comment (Atermonera) [inline code/modules/reagents/reagents/food_drinks.dm:4520]:
```suggestion
	description = "Juice from a ga'uli pod, used in skrellian and teshari cooking."

**Review by Atermonera — CHANGES_REQUESTED** [review]
Lore PRs relating to skrell mean I might actually be particular about syntax and grammar.

File: code/modules/food/food/snacks.dm [hunk 5]
```diff
@@ -7522,3 +7524,287 @@
...
+		if(6 to 10)
+			icon_state = "neaera_candied_eyes10"
+		if(11 to 15)
+			icon_state = "neaera_candied_eyes15"
+		if(20 to INFINITY)
+			icon_state = "neaera_candied_eyes20"
+
+/obj/item/reagent_containers/food/snacks/neaeracandy
+	name = "candied neaera eye"
+	desc = "A candied neaera eye shaped into a cube. The mix of savoury and sweet is generally acceptable for most species, although the dish is not commonly liked due to the use of eyes."
```
[the remark this answers is not in the document]
Comment (Greenjoe12345) [reply code/modules/food/food/snacks.dm]: This item's referring to a single candy, so it referring to a single one is right, I'd think?

**Review by Atermonera — APPROVED** [review]

**Pull request merged**